platform/integrations/karpenter.mdx

```diff
@@ -0,0 +1,449 @@
+```yaml
+sync:
+  toHost:
+    pods:
+      enforceTolerations:
+      - vclusterID=<project-name>:NoSchedule
+```
+3. Create a node pool with the taint corresponding to the toleration in step 2, e.g.:
+
+<Tabs
+  defaultValue="EKS"
+  values={[
+    { label: "Provision NodePool for EKS", value: "EKS" },
+    { label: "Provision NodePool for AKS", value: "AKS" },
+  ]}
+>
+  <TabItem value="EKS">
+
+    ```bash
+    cat <<EOF | envsubst | kubectl apply -f -
+    apiVersion: karpenter.sh/v1beta1
+    kind: NodePool
+    metadata:
+      name: <name-of-your-choosing>
+    spec:
+      template:
+        spec:
+          taints:
+          - key: <project-name>
+            value: "true"
+            effect: NoSchedule
+          requirements:
+            - key: kubernetes.io/arch
+              operator: In
+              values: ["amd64"]
+            - key: kubernetes.io/os
+              operator: In
+              values: ["linux"]
+            - key: karpenter.sh/capacity-type
+              operator: In
+              values: ["spot"]
+            - key: karpenter.k8s.aws/instance-category
+              operator: In
+              values: ["c", "m", "r"]
+            - key: karpenter.k8s.aws/instance-generation
+              operator: Gt
+              values: ["2"]
+          nodeClassRef:
+            apiVersion: karpenter.k8s.aws/v1beta1
+            kind: EC2NodeClass
+            name: <node-class-name>
+      limits:
+        cpu: 100
+      disruption:
+        consolidationPolicy: WhenUnderutilized
+        expireAfter: Never
+EOF
+```
+
+  </TabItem>
+  <TabItem value="AKS">
+
+    ```bash
+    cat <<EOF | envsubst | kubectl apply -f -
+    apiVersion: karpenter.sh/v1beta1
+    kind: NodePool
+    metadata:
+      name: <name-of-your-choosing>
+    spec:
+      template:
+        spec:
+          taints:
+          - key: <project-name>
+            value: "true"
+            effect: NoSchedule
+          requirements:
+            - key: kubernetes.io/arch
+              operator: In
+              values: ["amd64"]
+            - key: kubernetes.io/os
+              operator: In
+              values: ["linux"]
+            - key: karpenter.sh/capacity-type
+              operator: In
+              values: ["on-demand"]
+            - key: karpenter.azure.com/sku-family
+              operator: In
+              values: [D]
+          nodeClassRef:
+            name: <node-class-name>
+      limits:
+        cpu: 100
+      disruption:
+        consolidationPolicy: WhenUnderutilized
+        expireAfter: Never
+EOF
+    ```
+
+  </TabItem>
+</Tabs>
+
+4. Create project that only allows template(s) with set tolerations. Go to project dropdown -> New Project -> Allowed Templates. Then clear all templates and select the template(s) that include tolerations for the node pool created in step 3. Now all pods created in vClusters belonging to the created project will deploy to designated node pool(s).
```

Comment:
Create a new project that only allows template(s) with set tolerations. Go to project dropdown and create a new project. In the `Allowed Templates` section, remove the `All Templates` option. Select the template(s) that include tolerations for the node pool created in step 3. Now, all pods created in virtual clusters belonging to this project will deploy to designated node pool(s).